docs/components/nav.vue

```diff
@@ -60,9 +60,6 @@
           right>
           <b-dropdown-item href="http://dev.bootstrap-vue.netlify.com">Development</b-dropdown-item>
```

Comment:
Shouldn't we use `https://` here because there is a redirect anyway.
A SSL cert for this subdomain would be ideal.

Reply:
Yeah, although I don't believe this link actually works atm, and will need to be updated before release.